test/Sentry.Tests/HubTests.CaptureEvent_ActiveTransaction_UnhandledExceptionTransactionEndedAsCrashed.Net4_8.verified.txt

```diff
@@ -50,6 +50,16 @@
               }
             ],
             DebugImages: [
+              {
+                Type: pe_dotnet,
+                ImageAddress: null,
+                ImageSize: null,
+                DebugId: ________-____-____-____-____________-_,
+                DebugChecksum: null,
+                DebugFile: mscorlib.pdb,
+                CodeId: ______________,
+                CodeFile: .../mscorlib.dll
+              },
```

Comment:
Hey look - now that are collecting it, we also get a debug id for mscorlib on .NET Framework. (This should light up some framework symbolication.)